core/src/db/merge/MergeApuStrategy.cpp

```diff
@@ -0,0 +1,37 @@
+//  Copyright (C) 2019-2020 Zilliz. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+//  with the License. You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software distributed under the License
+//  is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+//  or implied. See the License for the specific language governing permissions and limitations under the License.
+//
+
+#include "db/merge/MergeApuStrategy.h"
+
+#include <algorithm>
+
+namespace milvus {
+namespace engine {
+
+Status
+MergeApuStrategy::RegroupFiles(meta::FilesHolder& files_holder, MergeFilesGroups& files_groups) {
+    meta::SegmentsSchema& files = files_holder.HoldFiles();
+
+    // arrange files by ascending creation date order
```

Comment:
Could you tell me why APU require "arrange files by ascending creation date order"?

Reply:
Hi , 

in order to keep the data in rv files and the data loaded to the APU synchronized . 
APU records is been loaded according to their creation order. 
we didnt find a merge strategy that doing sort by creation data. 

for example : 
Records is added to a Milvus collection and been loaded to the apu. 
The next added records is been added to the APU in the end of the loaded datset.